src/resonanceReconstruction/src/phaseShift.hpp

```diff
@@ -1,17 +1,17 @@
-inline double phaseShift( const int l, const double channelRatio ){
-  if ( not l ){ return channelRatio; }
+inline double phaseShift( const int L, const double channelRatio ){
+  if ( L == 0 )
+    return channelRatio;
 
-  const auto rho2 = channelRatio * channelRatio;
-  const auto offset = [&]{
-    switch( l ){
-    case 1: return channelRatio;
-    case 2: return channelRatio * 3. / ( 3. - rho2 );
-    case 3: return channelRatio * ( 15. - rho2 ) / ( 15. - 6. * rho2 );
-    case 4: return channelRatio * ( 105. - 10. * rho2 )
-                                  / ( 105. + rho2 * ( -45. + rho2 ) );
-    default: throw std::exception();
+  const double rho2 = channelRatio * channelRatio;
+  const double offset = [&]{
```

Comment:
It's kind of weird that this is a lambda. Since you've been trying to get rid of lambdas, is there a reason why you kept this one?

Reply:
Agreed, I too thought it was kinda weird to use a lambda in this way. Interesting as well; I haven't seen a lambda used in that way. I'm tempted to replace with ternary conditional, but on the other hand, I understand that you're not really a fan of ternary conditional. One convenient thing about the lambda is that the (non-return) `throw` case fits in more conveniently than it would with ternary conditional, which requires that all possible selections actually involve a return.

Reply:
Unrelated remark: You may have noticed that here, and in my earlier PR as well, I changed constants like `105.` (the double 105) to `105` (the integer 105). In the cases where I've done that, the integer always immediately mixes with a double, and will thus be converted automatically by the compiler. No need, therefore, for the dot. In my opinion, these changes make the code a bit more readable.